packages/health-check/src/checks/critical.js

```diff
@@ -63,6 +80,25 @@ async function uploadCheck(done) {
   done({ name: "upload_file", time: calculateElapsedTime(time), ...data });
 }
 
+// skylinkHealthCheck checks if the skylink has reached full redundancy
+async function skylinkHealthCheck(skylink, numRetries, authCookie) {
+  try {
+    const response = await got(`https://${process.env.PORTAL_DOMAIN}/skynet/health/skylink/${skylink}`, {
+      headers: { cookie: authCookie },
+    });
+    const healthData = getResponseContent(response);
+    if (healthData.basesectorredundancy !== 10 && numRetries > 0) {
+      return skylinkHealthCheck(skylink, numRetries - 1);
```

Comment:
This will run X times without any delay between the calls, we probably want to add some sleep time between calls. You'll have to assume some time constraint we want to give the server for a file to reach full redundancy.

Reply:
yea this is what I had done, but I removed it because I was hoping there was a nocache option.

Reply:
adding sleep back in